Source/VMware.vSphereDSC/Tests/Integration/Configurations/vCenterSettings/vCenterSettings_Config.ps1

```diff
@@ -96,6 +98,23 @@ Configuration vCenterSettings_WithTaskMaxAge_Config
     }
 }
 
+Configuration vCenterSettings_WithMotdAndIssue_Config
```

Comment:
The configuration looks good, please just update the brackets to follow the newly added Formatting Guidelines (Move the Configuration and Node opening bracket to be on the same line).